### ISSUE TITLE
Python is missing String instance methods

### ISSUE THREAD

**Issue by UltCombo** [codecombat/codecombat#3585]
Test case:

```py
hero.say('abc'.find(b) == 1) # expected: true, actual: runtime error
```

It seems like most of Python's String instance methods are missing. 

Python strings are parsed as JavaScript strings, right? I believe the easy way to implement the missing String methods would be to augment `String.prototype` from Filbert's runtime.

The more "correct" way would be to parse Python strings into custom objects, but that is a lot more of work and would certainly break a lot of existing users' code.

**Comment by nwinter** [comment]
Yeah, we just need to implement more methods [in Filbert](https://github.com/differentmatt/filbert).

**Comment by basicer** [comment]
The standard library for Codecombat python lives here now.

https://github.com/codecombat/skulpty/blob/master/lib/stdlib.js

If you'd like to help out by improving it, send a pull request to that repository.
